runtime/metricsview/astexpr.go

```diff
@@ -95,7 +96,7 @@ func (b *sqlExprBuilder) writeSubquery(sub *Subquery) error {
 		Offset:              nil,
 		TimeZone:            outer.TimeZone,
 		Label:               false,
-	}
+	} //exhaustruct:enforce
```

Comment:
Why is this separately required ?
Also if you take a pull from main. This is required at few more places introduced in the `MetricsViewSearch` PR.